src/crypto/cryptopp.cpp

```diff
@@ -117,6 +117,42 @@ void SymmCipher::cbc_decrypt(byte* data, unsigned len, const byte* iv)
 }
 
 /**
+ * @brief Encrypt symmetrically using AES in CBC mode and pkcs padding
+ *
+ * The size of the IV is one block in AES-128 (16 bytes).
+ *
+ * @param data Data to be encrypted
+ * @param iv Initialisation vector.
+ * @param result Encrypted message
+ * @return Void.
+ */
+void SymmCipher::cbc_encrypt_pkcs_padding(string *data, const byte *iv, string *result)
+{
+    aescbc_e.Resynchronize(iv ? iv : zeroiv);
```

Comment:
I'm not *that* familiar with C++, but to me it seems that `Resynchronize` is a method/function. It would be good to start them with a lower case letter for consistency like the rest (we're not dealing with MS Windows naming conventions here).